src/OrchardCore.Modules/OrchardCore.Users/OrchardCore.Users.csproj

```diff
@@ -18,7 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="AspNet.Security.OpenIdConnect.Primitives" />
+    <PackageReference Include="AspNet.Security.OpenIdConnect.Primitives" PrivateAssets="All" />
```

Comment:
@sebastienros FYI this package is only used for its constants (it's a recent change) so it doesn't make sense to have it as a NuGet dependency in the final .nuspec. `PrivateAssets="All"` should prevent that.